test/bin/mirror_registry.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=test/bin/common.sh
+source "${SCRIPTDIR}/common.sh"
+
+QUAY_VERSION=1.3.9
+REGISTRY_HOST=${REGISTRY_HOST:-$(hostname):8443}
+REGISTRY_ROOT=${REGISTRY_ROOT:-${HOME}/mirror-registry}
+REGISTRY_CONTAINER_DIR=${REGISTRY_CONTAINER_DIR:-${REGISTRY_ROOT}/containers}
+REGISTRY_CONTAINER_LIST=${REGISTRY_CONTAINER_LIST:-${REGISTRY_ROOT}/mirror-list.txt}
+PULL_SECRET=${PULL_SECRET:-${HOME}/.pull-secret.json}
+
+get_container_images() {
+    containers=""
+    local -r release_info_rpm=$(find "${IMAGEDIR}/rpm-repos" -name "microshift-release-info-*.rpm" | sort)
+    if [ -z "${release_info_rpm}" ] ; then
+        echo "Error: missing microshift-release-info RPMs"
+        exit 1
+    fi
+    for package in ${release_info_rpm}; do
```

Comment:
It would be useful for this loop to print some basic information about what it's doing. Just a line about "getting image references from $package" is probably enough.

Reply:
Done